radar/2023-09-12/klipfolio.md

```diff
@@ -5,9 +5,6 @@ quadrant:   methods-and-patterns
 tags:       [data-use]
 ---
 
-Klipfolio is a paid third-party SAAS option for building dashboards.
+[Klipfolio](https://www.klipfolio.com/) is a paid third-party SAAS option for building dashboards.
```

Comment:
I think we should move klipfolio out of adopt and into "stop", and add a recommendation to use Superset instead.

Reply:
Good point! I will make the same change in the Contributors' tech radar too.